src/panels/config/automation/manual-automation-editor.ts

```diff
@@ -324,6 +328,9 @@ export class HaManualAutomationEditor extends LitElement {
     // deselect previous selected row
     this._sidebarConfig?.close?.();
     this._sidebarConfig = ev.detail;
+    setTimeout(() => {
```

Comment:
Why is the `setTimeOut`? Don't you want something like this?:

```ts
const sidebar = this._sidebarElement;
await sidebar.updateComplete;
sidebar.focus();
```

Reply:
or even better, move the `updateComplete` waiting to the focus function of `ha-automation-sidebar`